### ISSUE TITLE
Courier pricing - Automate process

### ISSUE THREAD

**Issue by Jer0men** [dangarfield/eve-jita-anywhere#6]
The prices don't seem to match with prices I'd expect to pay for a trip from Jita to Dodixie. 

I'd expect to see the lowest price available.

So:
Rush - 80mil from Red Frog
Normal - 14.4mil from PushX

Also when looking at the Delivery Fee and Rush Fee they appear to be being added together in the final Delivery Fee value. 

![image](https://github.com/dangarfield/eve-jita-anywhere/assets/32482382/46814222-80d9-4453-94cf-a444c4244db1)

**Comment by dangarfield** [comment]
Good one. 

Shipping fees - I mimicked the pushx API, but not in an automated fashion (the PushX API is just too slow to be reasonable). This is area improvement, I thought about it and added it. If you're happy, I'll make sure that it updates automatically when PushX changes. If you know someone in PushX who can give me a non rate-limited API, even better, but until then, we have to mimick it.

In this specific instance, the rush fee changed from 100m to 75m. I've set this to 75m, but I'll leave this bug open so that I can automate the process.
